<launch>
    <arg name="name" default="micras_node"/>
    <arg name="debug" default="false"/>

    <include file="$(find-pkg-share micras_simulation)/launch/include/world.launch.xml"/>

    <include file="$(find-pkg-share micras_simulation)/launch/include/node.launch.xml">
        <arg name="name" value="$(var name)"/>
        <arg name="debug" value="$(var debug)"/>
    </include>
</launch>
